packages/bpk-component-breakpoint/src/BpkBreakpoint-test.tsx

```diff
@@ -22,99 +22,111 @@ import ReactDOMServer from 'react-dom/server';
 import { BREAKPOINTS } from './BpkBreakpoint';
 
 describe('BpkBreakpoint', () => {
-  it('should render if the breakpoint is matched', () => {
-    jest.resetModules();
-    const BpkBreakpoint = require('./BpkBreakpoint').default; // eslint-disable-line global-require
-    jest.mock('./useMediaQuery', () => () => true);
-
-    const { asFragment } = render(
-      <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
-        {(matches: boolean) =>
-          matches ? <div>matches</div> : <div>does not match</div>
-        }
-      </BpkBreakpoint>,
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
-
-  it('should render if the breakpoint is not matched', () => {
-    jest.resetModules();
-    const BpkBreakpoint = require('./BpkBreakpoint').default; // eslint-disable-line global-require
-    jest.mock('./useMediaQuery', () => () => false);
-
-    const { asFragment } = render(
-      <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
-        {(matches: boolean) =>
-          matches ? <div>matches</div> : <div>does not match</div>
-        }
-      </BpkBreakpoint>,
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
-
-  describe('SSR mode', () => {
-    beforeEach(() => {
+  describe('children as component', () => {
```

Comment:
BpkBreakpoint responsibilities have changed. It now only takes the match response from `useMediaQuery()` and works out whether to:

- render the children
- call the children function